attribute_set/views/attribute_option_view.xml

```diff
@@ -12,7 +12,7 @@
                     <field
                         name="value_ref"
                         colspan="2"
-                        groups="base_custom_attribute.group_advanced_attribute"
+                        groups="attribute_set.group_advanced_attribute"
```

Comment:
do not prefix with the module name. AS we are in the module it will be automatically prefixed before the search

Reply:
When I try to remove these module prefix on security groups, I got some :
```
WARNING db odoo.addons.base.models.ir_ui_view: The group group_advanced_attribute defined in view attribute.option.form.popup does not exist! 
...
KeyError: ('res.users', <function Users._has_group at 0x7f2742a56840>, 1, 'group_advanced_attribute')
...
  File "/odoo/src/odoo/addons/base/models/res_users.py", line 741, in _has_group
    assert group_ext_id and '.' in group_ext_id, "External ID must be fully qualified"
AssertionError: External ID must be fully qualified
```

So it looks like I have to let it on groups... is it right?